.github/workflows/docker.yml

```diff
@@ -0,0 +1,60 @@
+name: Docker Build
+
+on:
+  pull_request:
+    branches:
+      - main
+      - test
```

Comment:
These won't be needed in the future and can be removed, right?

Reply:
Done.